source/libraries/ContractInterfaces.ts

```diff
@@ -903,6 +903,27 @@
     }
 
 
+    export class GethContract extends Contract {
```

Comment:
Feels like this should not be included (just for testing)?

Reply:
Ah this shouldn't be there.